integration_testing/features/coach/coach-create-lesson.feature

```diff
@@ -142,11 +162,11 @@ Feature: Coach creates lessons
     When I click an exercise <exercise> card
     Then I am on the preview page for exercise <exercise>
     When I click the *Add* button
-    Then I see a snackbar confirmation *1 resource added to the lesson*
+    Then I see a snackbar confirmation *Added 1 resource to the lesson*
       And I see the *Added* label 
       And I see the *Remove* button
     When I click the *Remove* button
-    Then I see a snackbar confirmation *1 resource removed from the lesson*
+    Then I see a snackbar confirmation *Removed 1 resource from the lesson*
```

Comment:
a thought:

We could make gherkin stories a little easier to maintain by avoiding quoting specific messages. For example this could say

>  Then I see a snackbar confirmation that the resource was removed from the lesson

Reply:
For Gherkin scenarios, yes I agree we can make it easier to maintain.  

The specific texts are useful for automated unit tests coz some text changes may have effect on other things like translation/s of texts.

Reply:
True, and I wasn't consistent with spelling all of them them out 😊 

But being cognizant of the actual wording is important for the same i18n reason Cyril mentioned, although that is valid mostly for me, and not that much for the rest of MrPau's team.